exporter/collector/metrics_test.go

```diff
@@ -586,6 +586,7 @@ func TestSummaryPointToTimeSeries(t *testing.T) {
 	quantile.SetQuantile(1.0)
 	quantile.SetValue(1.0)
 	end := start.Add(time.Hour)
+	point.SetStartTimestamp(pdata.NewTimestampFromTime(start))
```

Comment:
should we add a test for the 0 value conversion?

Reply:
Zero is the default value, so it was being implicitly used in one of the tests.  I made that usage explicit in the most recent commit.

Reply:
Ah, sounds good to me. Think I get it now